projects/1.20/assets/slashblade/slashblade/lang/zh_cn.json

```
@@ -0,0 +1,179 @@
+{
+  "itemGroup.slashblade": "拔刀剑",
+  "itemGroup.slashblade.desc": "堕入阿修罗之道",
+
+  "attribute.name.generic.reachDistance":"攻击距离",
+
+  "stat.slashblade.sword_summoned": "召唤幻影剑",
+  "entity.slashblade.abstract_summoned_sword": "幻影剑",
+
+  "slashblade.tooltip.killcount": "杀敌数：%1$s",
+  "slashblade.tooltip.refine": "强化等级：%1$s",

```

```suggestion
  "slashblade.tooltip.refine": "重铸次数：%1$s",
```
实际玩下来，这玩意还得叫重铸
至于是`重铸次数`还是`重铸等级`得看你了

---

重铸次数吧 ，等级感觉比较容易模糊